### PR TITLE
update examples: add montioring and add ksn proxy

### PR DESCRIPTION
**Pull request by ericsyh**

### CODE REVIEW

**Comment by ericsyh** [comment]
@tuteng @urfreespace PTAL.

File: operation/auto-scaling/custom-metrics.yaml [hunk 1]
```diff
@@ -6,7 +6,11 @@ metadata:
   labels:
     k8s.streamnative.io/coordinator-name: private-cloud
 spec:
+<<<<<<< HEAD:autoscaling/scalingBehavior.yaml
```
Comment (tuteng) [inline operation/auto-scaling/custom-metrics.yaml:9]: Please fix conflict

File: operation/auto-scaling/custom-metrics.yaml [hunk 2]
```diff
@@ -6,7 +6,11 @@ metadata:
...
+  image: streamnative/private-cloud:3.1.1.1
+=======
   image: streamnative/private-cloud:3.1.0.5
+>>>>>>> origin/main:operation/auto-scaling/custom-metrics.yaml
```
Comment (tuteng) [inline operation/auto-scaling/custom-metrics.yaml:13]: Please fix conflict

File: protocols/ksn.yaml [hunk 3]
```diff
@@ -1,12 +1,24 @@
...
     k8s.streamnative.io/coordinator-name: private-cloud
 spec:
+<<<<<<< HEAD
+<<<<<<<< HEAD:protocols/ksn.yaml
+  image: streamnative/private-cloud:3.1.1.1
+========
   image: streamnative/private-cloud:3.1.0.5
+>>>>>>>> origin/main:protocols/aop.yaml
+=======
+  image: streamnative/private-cloud:3.1.0.5
```
Comment (tuteng) [inline protocols/ksn.yaml:20]: Please fix conflict

File: storage/zk-storage.yaml [hunk 4]
```diff
@@ -1,12 +1,16 @@
...
 metadata:
-  name: zookeepers
+  name: private-cloud
   namespace: pulsar
   labels:
     k8s.streamnative.io/coordinator-name: private-cloud
 spec:
+<<<<<<< HEAD
+  image: streamnative/private-cloud:3.1.1.1
+=======
```
Comment (tuteng) [inline storage/zk-storage.yaml:11]: Please fix conflict

**Comment by ericsyh** [comment]
@tuteng PTAL for the update.

**Review by tuteng — APPROVED** [review]

**Pull request merged**